learning/imitation/iil-dagger/README.md

```diff
@@ -1,28 +1,43 @@
-# Imitation Learning using Dataset Aggregation
+# Imitation Learning
 
 ## Introduction
-In this baseline we train a small squeezenet model on expert trajectories to simply clone the behaviour of the expert.
-Using only the expert trajectories would result in a model unable to recover from non-optimal positions ,Hence we use a technique called DAgger a dataset aggregation technique with mixed policies between expert and model.
+In this baseline we train a small squeezenet model on expert trajectories to simply clone the behavior of the expert.
+Using only the expert trajectories would result in a model unable to recover from non-optimal positions,Hence we use a technique called DAgger a dataset aggregation technique with mixed policies between expert and model.
```

Comment:
Using only the expert trajectories would result in a model unable to recover from non-optimal positions; Instead, we use a technique called DAgger: a dataset aggregation technique with mixed policies between expert and model.